.github/scripts/nft_boost/activateBoost.ts

```diff
@@ -0,0 +1,226 @@
+import { Address, getAddress } from 'viem'
+import * as fs from 'fs'
+import { config as envConfig } from 'dotenv'
+import axios from 'axios'
+import { BackersManagerAbi, BuilderRegistryAbi, GaugeAbi } from '../../../src/lib/abis/v2'
+
+// constant address. Calculation based on sc: address public constant COINBASE_ADDRESS = address(uint160(uint256(keccak256("COINBASE_ADDRESS"))));
+const rewardCoinbaseAddress = getAddress('0xf7aB6CfaebbADfe8B5494022c4C6dB776Bd63b6b')
+
+const boostDataFolder = `nft_boost_data`
+const nftActiveBoostPath = `${boostDataFolder}/latest`
+
+const [, , ...args] = process.argv
+
+type Args = {
+  nftContractAddress: Address
+  boostPercentage: number
+  env: string
+}
+
+interface NFTEvent {
+  address: string
+  blockNumber: string
+  data: string
+  gasPrice: string
+  gasUsed: string
+  logIndex: string
+  timeStamp: string
+  topics: string[]
+  transactionHash: string
+  transactionIndex: string
+}
+
+async function getActions() {
+  const { publicClient } = await import(`../../../src/lib/viemPublicClient`)
+  const builderRegistryAddress = process.env.NEXT_PUBLIC_BUILDER_REGISTRY_ADDRESS
+  const backersManagerAddress = process.env.NEXT_PUBLIC_BACKERS_MANAGER_ADDRESS
+  const chainId = process.env.NEXT_PUBLIC_CHAIN_ID
+  const rifWalletServicesUrl = process.env.NEXT_PUBLIC_RIF_WALLET_SERVICES
+
+  const getLatestBlockNumber = async (): Promise<bigint> => {
+    return await publicClient.getBlockNumber()
```

Comment:
NIT: we could remove async/await and directly return `publicClient.getBlockNumber()`